docker/manage

```diff
@@ -218,6 +222,28 @@ configureEnvironment() {
   export KEYCLOAK_LOGLEVEL="WARN"
   export KEYCLOAK_ROOT_LOGLEVEL="WARN"
 
+
+  # wallet-db
+  export WALLET_TYPE="postgres_storage"
+  export WALLET_ENCRYPTION_KEY="key"
+  export POSTGRESQL_WALLET_HOST="wallet-db"
+  export POSTGRESQL_WALLET_PORT="5432"
+  export POSTGRESQL_WALLET_DATABASE="wallet_db"
+  export POSTGRESQL_WALLET_USER="walletuser"
+  export POSTGRESQL_WALLET_PASSWORD="walletpassword"
+  export POSTGRESQL_WALLET_ADMIN_USER="postgres"
+  export POSTGRESQL_WALLET_ADMIN_PASSWORD="mysecretpassword"
+
+  # ACA-py Dev
+  export AGENT_NAME="vc-oidc-controller-agent"
+  export GENESIS_URL="${GENESIS_URL:-http://test.bcovrin.vonx.io/genesis}"
+  export AGENT_SEED="000000000000000000000000Steward1"
```

Comment:
Could have the seed be randomly generated - not an issue since this agent will be read-only, but a possibility. See [this](https://github.com/bcgov/issuer-kit/blob/main/docker/manage#L463) as an example.